net/znc/files/znc.init

```diff
@@ -12,7 +15,6 @@ DISABLED=
 
 RUNAS_USER=
 RUNAS_GROUP=
-RUNAS_SHELL=
```

Comment:
My understanding of this is that 'su' is used to switch user.  If that user doesn't have a valid shell then the 'su' will fail.
I tested znc with the 'shell' module and znc user shell set to /bin/false and all was well....although why on earth you'd give shell access via irc!!!!

Don't forget that this change also removes a dependency on 'su' being installed.